pkg/tfgen/generate.go

```diff
@@ -456,6 +460,10 @@ func (g *Generator) makeObjectPropertyType(typePath paths.TypePath,
 		// This seems wrong, so we ignore the second return value here for now.
 		doc, _ := getNestedDescriptionFromParsedDocs(entityDocs, objPath.join(key))
 
+		// If we have no result from entityDocs, we look up the TF schema Description.
+		if doc == "" {
```

Comment:
I would love to see a test that includes elided text from a `.Description()`, and then shows that our elision and replace logic applies to `.Description()` based descriptions. I'm pretty sure that neither currently apply, since their application are tied into parsing `entityDocs`.

Untangling the replace logic will be painful, so I don't believe it is worth blocking this PR. I think we need to make sure that [`reformatText`](https://github.com/pulumi/pulumi-terraform-bridge/blob/c85c6b160505d6ababf9d898204e88c7df8292c4/pkg/tfgen/docs.go#L1967-L1969) is called on `.Description()` based properties.

Looking at https://github.com/pulumi/pulumi-docker/compare/master...nested-configs-sample-schema, we can actually see several instances where normally elided text was added.

Reply:
Yeah looks like editing the test bridge *indirectly* covers the functionality, I can't recall do we have an assert somewhere that regenerating test bridge matches the checked in schema? We might have that..  So technically this is covered - but just to help our future selves I'd also love to see a simple dedicated test that asserts the desired description behavior.

Reply:
I created a dedicated test provider that ensures that these nested descriptions are modified as expected. The granular behavior of `reformatText` is already under test of course.